client/my-sites/domains/domain-management/list/style.scss

```diff
@@ -19,8 +19,14 @@
 	overflow: hidden;
 
 	.domain__primary-flag {
-		vertical-align: middle;
 		margin-left: 8px;
+		background: var( --sidebar-menu-hover-background );
+		color: var( --sidebar-menu-hover-color );
+		font-size: 12px;
+		border-radius: 12px;
+		display: inline-block;
+		padding: 2px 10px;
+		vertical-align: baseline;
```

Comment:
What's the reason for changing this to `baseline`? It looks more aligned to me when using `middle`:

| `baseline` | `middle` |
|---|---|
| <img width="276" alt="Screen Shot 2019-09-13 at 12 00 48" src="https://user-images.githubusercontent.com/1233880/64877018-83328580-d61e-11e9-91ac-42a9cf94c8e2.png"> | <img width="272" alt="Screen Shot 2019-09-13 at 12 01 25" src="https://user-images.githubusercontent.com/1233880/64877028-89286680-d61e-11e9-9f37-a46c43001b08.png"> |

Reply:
🤔 You're right here I didn't need to change this this property, it is better aligned with middle